collector/lib/ResourceSelector.cpp

```diff
@@ -0,0 +1,143 @@
+//
+// Created by Robby Cochran on 3/28/24.
+//
```

Comment:
Not to diminish Robby's contributions, but I think we want to remove this comment and the one on the header file. We can mention him in a commit message as a co-author.

Reply:
Done